docs/proposals/keadm-debug.md

```diff
@@ -0,0 +1,316 @@
+  runtime  Check whether the node Container runtime is normal, can use parameter `--runtime` to set container runtime,  default is docker
+  network  Check whether the node can communicate with the endpoint on the cloud,can use parameter `--ip` to set test ip, default to ping clusterdns
+  pid      Check if the current number of processes in the environment is too many. If the number of available processes is less than 5%, the number of processes is considered insufficient
+  
+Flags:
+  -h, --help   help for keadm debug check
+
+```
+
+### keadm debug collect --help
+
+```
+Obtain all the data of the current node, and then provide it to the operation and maintenance personnel to locate the problem
+
+Usage:
+  keadm debug collect [flags]
+
+Examples:
+
+keadm debug collect --path
+
+Flags:
+  --config  Specify configuration file, defalut is /etc/kubeedge/config/edgecore.yaml
+  --path    Cache data and store data compression packages in a directory that defaults to the current directory
+  --detail  Whether to print internal log output
+
+Flags:
+  -h, --help   help for keadm debug collect
+
+```
+
+### keadm debug get --help
+
+```
+Prints a table of the most important information about the specified resource from the local database of the edge node
+
+Usage:
+  keadm debug get [resource] [flags]
+
+Examples:
+
+# List all pod
+keadm debug get pod -A
+
+# List all pod in namespace test
+keadm debug get pod -n test
+
+# List a single configmap  with specified NAME
+keadm debug get configmap web -n default
+
+# List the complete information of the configmap with the specified name in the yaml output format
+keadm debug get configmap web -n default -o yaml
+
+Available resource:
+  pod
+  node
+  service
+  secret
+  configmap
+  endpoint
+
+Flags:
+  -h, --help            Help for keadm debug get
+  -A, --all-namespaces  List the requested object(s) across all namespaces
+  -n, --namespace=''    List the requested object(s) in specified namespaces
+  -o, --output=''       Output format. One of:json|yaml|jsonpath=...
+  -l, --selector=''     Selector (label query) to filter on, supports '=', '==', and '!='.(e.g. -l key1=value1,key2=value2)
+
+```
+
+## Explaining the commands
+
+### Worker Node (at the Edge) commands
+
+`keadm debug diagnose`
+
+- What is it?
+  
+- This command will be help to diagnose specific fault scenarios in an all-roundly way and locate the cause of the fault
+  
+- What shall be its scope ?
+    1. Use command `all` can diagnose all resource
+    2. Use command `node` can troubleshoot the cause of edge node failure with installed software
+       1. check system resources is enough
+       2. check container runtime is running 
+       3. check all edgecore components are running
+       4. check cloudercore can be connected
+    3. Use command `pod` can troubleshooting specific container application instances on nodes
+       1. check pod Is the configuration correct 
```

Comment:
It is not a good place to check pod configuration.

Reply:
ok, I will removed this item